src/pages/components/autocomplete.mdx

```diff
@@ -1,10 +1,110 @@
 ---
 title: Autocomplete
 description: >
-  Description will be here
+  Autocomplete is an input field that filters results as users type. This helps users find
+  something quickly in a large list of options.
+package: '@zendeskgarden/react-dropdowns'
+components:
+  - dropdowns/src/elements/Autocomplete/Autocomplete.tsx
+  - dropdowns/src/elements/Dropdown/Dropdown.tsx
+  - dropdowns/src/elements/Menu/Menu.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - dropdowns/src/elements/Fields/Field.tsx
+  - dropdowns/src/elements/Fields/Hint.tsx
+  - dropdowns/src/elements/Fields/Label.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To filter down a large list of options</li>
+      <li>To quickly find a known option</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To make more than one selection use <a href="./multiselect">Multiselect</a> instead
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The basic usage of an Autocomplete component.
```

Comment:
@vsri1137 Looks like we need an editing pass here (this is a fragment)

Reply:
@allisonacs On inspection, I'm not quite sure what this text is meant to convey. From the image my naive guess is: the basic usage of an Autocomplete involves an Input combined with a Dropdown.

Is it something like that?

Reply:
It seems that there are two other places where the "Default" copy is similar. Tabs and Multiselect:

### Tabs 

> This is the basic usage of a Tab component.

### Multiselect

> The default usage of a Multiselect component.

Reply:
However other "Default" copy is structured a little differently:

### Anchor 

> The Anchor component is a styled `<a>` tag. It accepts all standard anchor attributes signals a way
to navigate.

### Breadcrumb

> Breadcrumbs and the [Anchor](https://zendeskgarden.netlify.app/components/anchor) component work together. Each element (a "crumb" if
you will) is separated from the others with an arrow.

### Checkbox

> A Checkbox’s label is part of its touch target.

### Radio

> A Radio’s label is part of its touch target.

### Input
> An Input is accompanied with a label indicating its purpose.


### Select
> Select works in combination with the Menu component to display available options.

Reply:
Since the default copy is already inconsistent, I think we should merge this PR, update the copy in Google Docs, and create a new PR to make the default copy consistent.